0-os/templates/influxdb_client/README.md

```diff
@@ -0,0 +1,45 @@
+## template: influxdb_Client
+
+### Description:
+This template is responsible for create influxdb client 
+
+### Schema:
+
+- `instanceName`: Name of the instance
```

Comment:
you don't need to have an instance name in the schema. use the name of the service itself as instance name of the client

Reply:
ok , but when i create database at statistics Template should get the name of service it self to get influxdb_client ?

Reply:
yes. So you create the influxdb_client service then you create the statistics one and in the data you give the name of the influxdb_client so it knows what to use